libs/nx-esbuild/src/executors/build/executor.ts

```diff
@@ -45,6 +45,6 @@ export default async function runExecutor(
     })
 
     return {
-        success: result.errors.length === 0,
+        success: !(result.errors && result.errors.length !== 0),
```

Comment:
`!result.errors?.length` could be a simpler statement, I think that would work as undefined and 0 are both falsy

Reply:
Works